### ISSUE TITLE
Cannot change severity of rule

### ISSUE THREAD

**Issue by apsMSIDC** [realm/SwiftLint#2904]
Xcode Version : 11.1
Download mode: Cocoapods
Version: 0.35.0
 
Found Related Issues on Github: Found but did'nt have proper answer

Trying to change the severity of a rule as follows:

```
vertical_whitespace:
- max_empty_lines: 3
- severity: error 
```

Error coming as : Invalid configuration for 'vertical_whitespace'. Falling back to default

**Comment by marcelofabri** [comment]
This is not the right syntax. `-` should be used for arrays, not for dictionaries. What you want is this:

```yml
vertical_whitespace:
  max_empty_lines: 3
  severity: error 
```
